go.mod

```diff
@@ -156,6 +156,8 @@ replace (
 	// TODO: remove when middleware will be implemented
 	github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.1-hooks
 
+	github.com/enigmampc/btcutil => github.com/scrtlabs/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25
```

Comment:
it use the same commit hash, how does it fix the vulnerability?

Reply:
I think the issue is mainly on the package path.

The user/organization of `github.com/enigmampc` changed its name to `github.com/scrtlabs`. Therefore `github.com/enigmampc` could be taken by others.

The code itself is not vulnerable, just the package path.

Reply:
![image](https://user-images.githubusercontent.com/86600130/149278835-92261e59-b3e7-4573-b724-841762a1c45c.png)

The information I got from security team. And they asked to replace the `github.com/enigmampc/btcutil` with `github.com/scrtlabs/btcutil`.

Reply:
github will redirect github.com/enigmampc/btcutil to github.com/scrtlabs/btcutil, while github.com/enigmampc would not be redirected to github.com/scrtlabs, this means the organization enigmampc can be registered by hackers to inject some malicious codes

Reply:
an opposite example is github.com/lazyledger which is github.com/Celestia now

Reply:
I see 👍

Reply:
Is there an upstream PR to fix it?

Reply:
Many modules in used will introduced this vulnerable dependency to `go.sum`.

If not doing the `replace` way, we may need to update a few dependencies: 
- `github.com/cosmos/ibc-go` to v2.0.2.
- `github.com/tharsis/ethermint` to v0.10.0-alpha1.
- `github.com/peggyjv/gravity-bridge/module`, we need to create a version on our own

Reply:
> Is there an upstream PR to fix it in dependency?

it'll be fixed if we upgrade ethermint dependency, but we haven't decided yet if we use ibc-go v3 or v2 yet.